docs/src/main/docs/guides/12_tracing_mp_guide.adoc

```diff
@@ -0,0 +1,759 @@
+across multiple services, and how to integrate with tracing with Kubernetes.  All examples use Zipkin and traces
+will be viewed using both the Zipkin API and UI.
+
+=== Create a sample Helidon MP project
+
+Use the Helidon MP Maven archetype to create a simple project that can be used for the examples in this guide.
+
+[source,bash,subs="attributes+"]
+.Run the Maven archetype:
+----
+mvn archetype:generate -DinteractiveMode=false \
+    -DarchetypeGroupId=io.helidon.archetypes \
+    -DarchetypeArtifactId=helidon-quickstart-mp \
+    -DarchetypeVersion={helidon-version} \
+    -DgroupId=io.helidon.examples \
+    -DartifactId=helidon-quickstart-mp \
+    -Dpackage=io.helidon.examples.quickstart.mp
+----
+
+[source,bash]
+.The project will be built and run from the `helidon-quickstart-mp` directory:
+----
+cd helidon-quickstart-mp
+----
+
+=== Setup Zipkin
+
+First, you need to run the Zipkin tracer.  Helidon will communicate with this tracer at runtime.
+
+[source,bash]
+.Run Zipkin within a docker container, then check the Zipkin server health:
+----
+docker run -d --name zipkin -p 9411:9411 openzipkin/zipkin  // <1>
+----
+<1> Run the Zipkin docker image named `openzipkin/zipkin`.
+
+
+[source,bash]
+.Check the Zipkin server health:
+----
+curl http://localhost:9411/health // <1>
+...
+{
+  "status": "UP", <2>
+  "zipkin": {
+    "status": "UP",
+    "details": {
+      "InMemoryStorage{}": {
+        "status": "UP"
+      }
+    }
+  }
+}
+----
+<1> Invoke the Zipkin REST API to check the Zipkin server health.
+<2> All `status` fields should be `UP`.
+
+=== Enable Tracing in your Helidon application
+
+Update the pom.xml file and add the following Zipkin dependency to the `<dependencies>`
+section (*not* `<dependencyManagement>`).
+
+[source,xml]
+.Add the following dependency to `pom.xml`:
+----
+<dependency>
+  <groupId>io.helidon.tracing</groupId>
+  <artifactId>helidon-tracing-zipkin</artifactId>
+</dependency>
+----
+
+All spans sent by Helidon to Zipkin need to be associated with a service.  Specify the service name below.
+
+[source,bash]
+.Add the following line to `META-INF/microprofile-config.properties`:
+----
+tracing.service=helidon-mp-1
+----
+
+[source,bash]
+.Build the application, skipping unit tests, then run it:
+----
+mvn package -DskipTests=true
+java -jar target/helidon-quickstart-mp.jar
+----
+
+[source,bash]
+.Run the curl command in a new terminal window and check the response:
+----
+curl http://localhost:8080/greet
+...
+{
+  "message": "Hello World!"
+}
+----
+
+
+=== Viewing tracing using Zipkin REST API
+
+Because you had tracing enabled, the previous `/greet` endpoint invocation resulted in a new trace being created.
+Let's get the trace data that was generated using the Zipkin API.  First, get the service information.
+
+NOTE: Helidon automatically enables tracing for JAX-RS resources methods so you don't need to use annotations with JAX-RS.
```

Comment:
Please add reference to MP OpenTracing, as that is the reason this is happening

Reply:
I have a refer to https://github.com/eclipse/microprofile-opentracing/releases/tag/1.3 at the bottom. Is there another reference?

Reply:
fixed